packages/core/src/auth-token-manager.ts

```diff
@@ -26,7 +26,6 @@ export type SecurityErrorCode = `Neo.ClientError.Security.${string}`
 /**
  * Interface for the piece of software responsible for keeping track of current active {@link AuthToken} across the driver.
  * @interface
- * @experimental Exposed as preview feature.
  * @since 5.8
```

Comment:
```suggestion
 * @since 5.14
```
Should this be when it was GA'd?

Reply:
That makes sense.